bridge/svix-bridge/src/config/mod.rs

```diff
@@ -238,5 +253,61 @@ impl WebhookReceiverConfig {
     }
 }
 
+#[derive(Clone, Deserialize)]
+#[serde(tag = "type", rename_all = "kebab-case")]
+pub enum PollerInputOpts {
+    SvixEvents {
+        app_id: String,
+        subscription_id: String,
```

Comment:
These settings are subject to change pending https://github.com/svix/svix-webhooks-private/pull/2484